src/NServiceBus.AcceptanceTests/Serialization/When_configuring_custom_xml_namespace.cs

```diff
@@ -10,7 +10,7 @@
 
     public class When_configuring_custom_xml_namespace : NServiceBusAcceptanceTest
     {
-        const string CustomXmlNamespace = "http://particular.net";
```

Comment:
I think that change is irrelevant because it could be any arbitrary URL here. But happy to leave it